draft-bar-cfrg-spake2plus.md

```diff
@@ -345,14 +362,11 @@ No IANA action is required.
 
 # Security Considerations
 
-SPAKE2+ appears in {{TDH}} along with a path to a proof that
-server compromise does not lead to password compromise under the DH assumption
-(though the corresponding model excludes pre-computation attacks).
+SPAKE2+ appears in {{TDH}} and is proven secure in {{UCAnalysis}}.
 
-Elements received from a peer MUST be checked for group membership:
-failure to properly validate group elements can lead to attacks. Beyond the cofactor
-multiplication checks to ensure that these elements are in the prime order subgroup
-of G, it is essential that endpoints verify received points are members of G.
+Beyond the cofactor multiplication checks to ensure that elements received from
+a peer are in the prime order subgroup of G, they also MUST be checked for group
+membership as failure to properly validate group elements can lead to attacks.
```

Comment:
Riad pointed out simpler text for this, captured in this issue: https://github.com/cfrg/draft-irtf-cfrg-hpke/issues/107